hack/boilerplate/boilerplate.py

```diff
@@ -88,6 +105,9 @@ def file_passes(filename, refs, regexs):
 
     # if we don't match the reference at this point, fail
     if ref != data:
+        logger.info('Header does not match reference, diff:')
+        for line in difflib.context_diff(ref, data, 'reference', 'actual'):
```

Comment:
I personally prefer unified diffs, as I'd guess do most people who use github.

Can you use the actual filename instead of 'actual'?
Can you make the out put directly usable in `patch` to fix the problem?